vendor/osqp_vendor/README.md

```diff
@@ -0,0 +1 @@
+# osqp_vendor
```

Comment:
I added a small reference to the source of this repo

```
added as `git` subtree from https://github.com/tier4/osqp_vendor to make `osqp` from https://github.com/oxfordcontrol/osqp available in ROS2.
```

Reply:
Thanks, added in f282c80